lib/iotagent-json.js

```diff
@@ -185,5 +185,20 @@ function stop(callback) {
     );
```

Comment:
CHANGES_NEXT_RELEASE entry should be included, about the modifications in this PR.

Reply:
Fixed 0069f53